helpers.go

```diff
@@ -173,3 +157,9 @@ func Find(slice []string, val string) (int, bool) {
 	}
 	return -1, false
 }
+
+func die(err error) {
```

Comment:
A die (or check or must) func is not really a "best practice" but it's easier to clean up later when you want to do "real" error handling than calling panic or whatever.